### DISCUSSION TITLE
v1.8.9: Changes in notification settings

### DISCUSSION THREAD

**Discussion by Futsch1** [Futsch1/medTimer#161] [Announcements]
v1.8.9 brings a major change in notification settings. These settings are unfortunately not compatible with the previous settings, so some configuration might have to be done again.

MedTimer now uses the built-in notification settings of Android to configure ringtones and general notification behavior. A direct link is provided from the settings menu.

<img src='https://github.com/Futsch1/medTimer/assets/63090558/90a25e1d-eda3-41de-ad71-35137aedef04' height=400 />

So the configuration of ringtone and disabling notifications needs to be done there.

In addition, a new feature was added that there are now two notification priority settings. A medicine can be configured to be either of standard or high priority. These two categories have different settings so that you can configure different ringtones, disable one of them or perform more in depth customization.

<img src='https://github.com/Futsch1/medTimer/assets/63090558/70e27e28-8c90-4f84-8a5b-6cc2200e221e' height=400 />

**Comment by Aque151** [comment]
I've just started using medTimer, and it's so far been the best and most helpful app I've tried for medication reminders. Quick question about Cycle Reminders- is there a way to have repeating cycles, and have the reminders send only on Active days?
For example, I have medication I take for 3 days straight, and then need to have a 2 day break from, before taking again for 3 days. I was able to put in the First Cycle Start, Active Days, Pause Days, however with the Remind On, the options are for days of the week, which unfortunately won't be helpful due to the cycle length.

It would be great to be able to set up a Repeating Cycle with the set number of Active Days and Pause Days, an End Date or Continuous, and then have an option for Remind On: Only Active Days.
(And just to make it extra fun, being able to set multiple dose reminders for it, such as 3 a day with reminders for those in the Active Days).

If there's already a way to do this that I've missed, apologies! 
It would be a huge help for me- it's a pain trying to keep track of which day I'm up to, did I take my two break days etc. and trying to plan ahead for things is a huge stress. 

Thanks for making this app, by the way. It's one less burden to carry regarding health issues. 
Take care!

**Reply by Futsch1** [reply]
Thanks for your feedback and your appreciation of my work.

Regarding your question: Active days would in your case be set to all days of the week (since your schedule is independent of the weekdays). I know that the setup is kind of puzzling here, especially since active days and cyclic reminders are independent from each other, but appear on the same screen. I might consider some alternative menu layout to make this more obvious.

So in your case, you would set the first cycle start to like 24th of June, have active days set to 3, pause days to 2 and active days keep all weekdays selected. This would mean a reminder on the 24th, 25th and 27th and then a pause of two days - regardless of the weekdays. This five day cycle would then repeat.

To make this more obvious, I will also add the upcoming reminder events to a calendar view I will start to work on soon, see #137

**Reply by Aque151** [reply]
Thank you for your reply! 
What I did for the medication that I take cyclically was (at least from what I can tell) is how you described- though my issue may have been that the cycle start was in the past, eg. I entered the cycle start as 24th June, and the real world date was 25th June at the time (as I was on day 2 of 3 of the cycle). Not sure if that may have effected it. Left all days selected for reminders, with Active Days 3 and Pause Days 2. From what I remember, I got reminders for that medication every day, regardless of if it was a break day.
What I'll do is when my next cycle is due to start, I'll put it in prior, and test if it happens again and report back what I find. 

Also quick question, and sorry if it's absolutely obvious and I've missed it somehow- is there a way to delete reminders for a medication but keep the medication itself? For instance I just took a short course of an antibiotic, and forgot to put in that there would only be 5 active days (or something equivalent) and I want to be able to delete the reminder, but keep the history of it for the sake of being able to track recent medications I've taken. I turned off all reminder days, but was wondering if there's any other way to outright delete any future reminders for it. 

Thanks so much again!

**Reply by Aque151** [reply]
Just read the conversation you linked- good to know I can delete it and it keeps the history!! Disregard my question! 😂 Sorry!
